### PR TITLE
Fix loading of preset title panel callback

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes:

```
PresetTitlePanel.js:128 Uncaught TypeError: this._onLoadedCallback is not a function
    at PresetTitlePanel._setupHtml (PresetTitlePanel.js:128:31)
    at HTMLDivElement.<anonymous> (PresetTitlePanel.js:58:100)
    at HTMLDivElement.<anonymous> (jquery.min.js:2:85383)
    at Function.each (jquery.min.js:2:3003)
    at S.fn.init.each (jquery.min.js:2:1481)
    at Object.<anonymous> (jquery.min.js:2:85365)
    at c (jquery.min.js:2:28327)
    at Object.fireWith [as resolveWith] (jquery.min.js:2:29072)
    at l (jquery.min.js:2:80045)
    at XMLHttpRequest.<anonymous> (jquery.min.js:2:82499)
```

@Benky please have a look.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by limonspb** [comment]
How to reproduce the bug before PR?

**Comment by haslinghuis** [comment]
Just enter the presets tab and look at console.

File: src/tabs/presets/TitlePanel/PresetTitlePanel.js
Comment (chmelevskij) [inline src/tabs/presets/TitlePanel/PresetTitlePanel.js:129]:
```suggestion
        if (typeof this._onLoadedCallback === 'function') {
```

Comment (haslinghuis) [reply src/tabs/presets/TitlePanel/PresetTitlePanel.js]: Fixed 👍

**Review by chmelevskij — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
